pkg/aws/metadata/handler_proxy_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2017 uSwitch
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package metadata
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+func TestProxyDefaultBlacklisting(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	allowedRoutes := regexp.MustCompile("^$")
```

Comment:
If we're not going to move this statement inside `newProxyHandler` I'd like the expression stored in a const for something like `WhitelistDefaultBlockEverything` so that it can be referenced in both the package and test.